### DISCUSSION TITLE
Discord Webhook [May 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2092] [Discord Threads]
> 🙋‍♂️ **INTEL**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### details for discord webhook are captured here   
#### https://docs.nocodb.com/developer-resources/webhooks  
#  
> 👨‍🦲 **dstala**  
#### if you still face issue, provide more details. we will look into.  
#  
> 🙋‍♂️ **INTEL**  
#### So no need to configure discord trough App store?  
#  
> 🙋‍♂️ **INTEL**  
#### I'm actualy stuck at configuring discor trough app store  
#  
> 👨‍🦲 **dstala**  
#### webhook, its 2 step process    
#### 1. configure credentials once in appstore.    
#### - click on 'install' on discord in appstore   
#### - store webhook link & give this webhook instance any generic name  
#  
> 👨‍🦲 **dstala**  
#### 2. open table for which you wish to configure webhooks, go to more > webhooks. Select trigger & action for associated trigger. details in link shared above  
#  
> 👨‍🦲 **dstala**  
#### https://docs.nocodb.com/developer-resources/webhooks/#discord  
#  
> 👨‍🦲 **dstala**  
#### step by step details here.  
#  
> 🙋‍♂️ **INTEL**  
#### I did that, I created a channel, created webhook, in app store clicked on install discord, added chanelname and pasted discord webhook, but when I click on test i get an error: Invalid Credetials  
#  
> 🙋‍♂️ **INTEL**  
#### Should I just ignore that and save?  
#  
> 👨‍🦲 **dstala**  
#### ignore that error for now.  
#  
> 👨‍🦲 **dstala**  
#### it need not be channel name- can be anything that you wish. its just a reference for us for this web-hook.  
#  
> 🙋‍♂️ **INTEL**  
#### K, gonna try  
#  
> 🙋‍♂️ **INTEL**  
#### Yeah, its working  
#  
> 🙋‍♂️ **INTEL**  
#### Should have just ignored error in app store  
#  
> 🙋‍♂️ **INTEL**  
#### thanks  
#  
> 👨‍🦲 **dstala**  
#### will raise issue to get 'web-hook' test fixed.  
#  
> 🙋‍♂️ **INTEL**  
#### Awsome, thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/972062612458176562)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
